libs9s/s9srpcclient.cpp

```diff
@@ -4533,7 +4537,14 @@ S9sRpcClient::createRedisSharded(
         jobData["redis_sharded_port"] = options->redisShardedPort();
 
     if (options->redisShardedBusPort() != 0)
+    {
+        if(version.contains("6"))
```

Comment:
What if there will be redis 7.6 or 7.4.6? Please change this condition to verify major number only